pkg/pkg.mk

```diff
@@ -4,15 +4,17 @@
 PKG_DIR?=$(CURDIR)
 PKG_BUILDDIR?=$(BINDIR)/pkg/$(PKG_NAME)
 
-download: $(PKG_BUILDDIR)/.downloaded
```

Comment:
Because `git-download` must be executed because it has (and should not have) no file equivalent.